datasources/src/main/scala/com/normation/plugins/datasources/UpdateHttpDataset.scala

```diff
@@ -137,7 +138,7 @@ class GetDataset(valueCompiler: InterpolatedValueCompiler) {
       // We only get the first element from the path, ignoring if there is several.
       // And if list is empty, returns "" (remove property).
       optJson.map {
-        case Nil        => DataSource.nodeProperty(datasourceName.value, ""   )
+        case Nil        => DataSource.nodeProperty(datasourceName.value, GenericProperty.parseValue("").getOrElse(ConfigValueFactory.fromAnyRef("")))
```

Comment:
you have `toConfigValue` for that - sorry for previous comment, I forgot that one was able to fail